prime-angular-frontend/src/app/shared/models/enrolment.model.ts

```diff
@@ -14,6 +14,8 @@ import { CareSetting } from '@enrolment/shared/models/care-setting.model';
 import { RemoteAccessLocation } from '@enrolment/shared/models/remote-access-location.model';
 import { RemoteAccessSite } from '@enrolment/shared/models/remote-access-site.model';
 import { OboSite } from '@enrolment/shared/models/obo-site.model';
+// This is needed for legacy enrolments
```

Comment:
Don't need this comment on the import.  Just wanted `Needed for legacy enrolments` in the models so we can add other keys under that line in the future, and know which ones are explicitly legacy keys.

Reply:
fixed